.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,16 +20,31 @@ Reviewing guidelines: https://hydra.nixos.org/job/nixpkgs/trunk/manual/latest/do
 
 ###### Things done
 
-<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->
+<!-- 
+Requirements: 
+- install `nixpkgs-review` for good with `nix-env -f '<nixpkgs>' -iA nixpkgs-review`
+- consult usage documentation: https://github.com/Mic92/nixpkgs-review#usage
+- setup github api token: https://github.com/Mic92/nixpkgs-review#github-api-token
```

Comment:
I think we can skip this step. The github token is only required for advanced features.

Reply:
I have a split argument about this:
- pull requester will be nudged into graduating into reviewers
- it helps towards a standardized review process if the pull requester already does: `nixpkgs-review post-result`

(and also: don't be too easy on pull requester, to shift the balance away from reviewers)

What do you think about those arguments?

Reply:
Could we make the request for API token part of first-time running nixpkgs-review? That way we would not have to mention it here.

Reply:
@jtojnar This is a really nice suggestion! I'll take it out here and open an issue to track it.